doc/unbound.conf.5.in

```diff
@@ -2127,6 +2127,13 @@ If this timeout expires Unbound closes the connection, treats it as
 if the Redis server does not have the requested data, and will try to
 re-establish a new connection later.
 This option defaults to 100 milliseconds.
+.TP
+.B redis-set-ttl: \fI<yes or no>
+If redis ttl is enabled. If yes, unbound sets ttl for redis records so
+that redis can evict keys that have expired automatically. If unbound
+is configured to serve expired entries and there is no expired ttl set,
+this option is internally reverted to "no".
+This option defaults to no.
```

Comment:
I would mention that support for SETEX is required for this to work (redis >= 2.0.0).